docs/DeveloperGuide.adoc

```diff
@@ -262,6 +279,208 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g user prefs file location, logging level) through the configuration file (default: `config.json`).
 
+== Features
+
+[[Features-Calendar]]
+
+TravEzy has several features which are listed below.
+Each of these features have their own design and implementation logic.
+
+=== Calendar
+
+The calendar feature in TravEzy allows users to easily find a free period of time to travel.
+
+*Calendar Class Diagram:*+
+
+image::CalendarClassDiagram.png[]
```

Comment:
Calendar should not have an extra arrow to abstract class events. 
Calendar can be placed below ReadOnlyCalendar to improve readability